3_javascript/javascript2/week3/The fastest presser in this realm.js

```diff
@@ -0,0 +1,41 @@
+let lCount = 0;
```

Comment:
Consider avoiding spaces in file names.  You could name it for example fastest_presser_game.js or something similar.